README.md

```diff
@@ -13,7 +16,25 @@ In this Project, I used HTML and CSS (Media, Transition, flexbox, Grid and more)
 
 ## Live Demo
 
-- [Capstone-Project](https://sediqullahbadakhsh.github.io/badakhsh-portfolio/).
+- [Capstone-Project](https://sediqullahbadakhsh.github.io/Capstone-project/).
+
```

Comment:
thank you @Felipeg005 , that's because I haven't merged it. when it gets approved and we should be able to see it.
since I have made all the required changes, don't you think I needed the approval. or do I really need to submit one last time? if you think there's nothing else to do with this, please do approve so that I can merge it with main branch and submit the project. @

Reply:
Ok, I understand, but in order to approve the capstone project the live version must be available and properly working, 
You can fix this by deploying the branch that has all the last updates in that way your project will be complete, and after the approval and merging you can change it to the main branch.

Sadly I can't approve after I sent request changes 😞 , please fix the live version and ask for a new review. And please consider checking if your live version is working properly when delivering your next projects.

Please remember to add the link to the Home page in the Desktop version before asking for a new review, I noted that it's missing too, so you will go straight forward to approval.

You are almost there @sediqullahbadakhsh , it was a pleasure reviewing your project 👍

Reply:
- [ ] [REQUIRED] You have done a great job deploying the capstone live version using github pages, however, not all images work both on the desktop and the mobile version.
To fix this, try removing the first forward slash before your image src/path. I hope this fixes the issue for you.
#### What you currently have is:
 "./assets/logos/Logo.png" alt="compnay logo" 

#### My recommended solution:
 "assets/logos/Logo.png" alt="compnay logo" 

#### This is what you currently have on the live page (Desktop)
![Screenshot from 2022-02-03 18-11-19](https://user-images.githubusercontent.com/58692590/152396104-f615e74e-d33d-4e9f-b14e-a894b10ab6cf.png)

#### This is what you currently have on the live page (Mobile)
![Screenshot from 2022-02-03 18-16-59](https://user-images.githubusercontent.com/58692590/152396431-7ac58c43-8bc1-4216-8cba-ce9907cc46f3.png)

Reply:
Hello @Whoistolu  thank you so much, I just removed the slash and it worked for me.

Reply:
- [ ] [REQUIRED] You have done a great job deploying your project to github pages, however, the images seems to be broken on both the desktop and the mobile version.

I have a fix for you:
Kindly remove the forward slash in front of your image source/path
#### what you currently have on your code-case:
"./assets/logos/Logo.png" alt="compnay logo"
#### My Recommendation:
"assets/logos/Logo.png" alt="compnay logo"
This is what the live version of your project currently looks like (Desktop)
![Screenshot from 2022-02-03 18-11-19](https://user-images.githubusercontent.com/58692590/152400532-0ddab933-091a-45f0-ac45-610ec4896422.png)

This is what the live version of your project currently looks like (Mobile Version)
![Screenshot from 2022-02-03 18-16-59](https://user-images.githubusercontent.com/58692590/152400704-94427557-9855-4653-ba9c-b3f6f1b0e5bd.png)

Reply:
> Hello @Whoistolu thank you so much, I just removed the slash and it worked for me.

You are welcome.
Happy coding!!!